controllers/todo.js

```diff
@@ -0,0 +1,84 @@
+const { Todo } = require('../models');
+
+class TodoController {
+    static main(req, res) {
+        Todo.findAll()
+            .then(data => {
+                res.status(200).send(data);
+            })
+            .catch(err => {
+                console.log(err);
+                res.status(500).json({ msg: 'Something went wrong!' });
+            })
+    }
+
+    static detail(req, res) {
+        Todo.findByPk(+req.params.id)
+            .then(data => {
+                res.status(200).send(data);
+            })
+            .catch(err => {
+                res.status(404).json({ msg: 'Not found' });
+            })
+    }
+
+    static getNew(req, res) {
+        res.render('new-todo');
+    }
+
+    static postNew(req, res) {
+        let data = {
```

Comment:
Mungkin gue lebih seneng yang lebih spesifik untuk penamaan data, karena ini data todo mungkin nama variable nya bisa dirubah juga dengan `todo` 
```suggestion
        let todo = {
```